CMakeLists.txt

```diff
@@ -4,7 +4,8 @@ project(libmill C)
 include(CheckSymbolExists)
 include(CheckFunctionExists)
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu11 -Wall -Wextra")
```

Comment:
Why is gnu11 necessary? I would prefer the library to work even with old compilers.